bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| sensorNameVoc6                   | vocControl      | Sensor name - VOC 6                                              | String                   |      |                                                                                                      |
+| sensorNameVoc7                   | vocControl      | Sensor name - VOC 7                                              | String                   |      |                                                                                                      |
+| sensorNameVoc8                   | vocControl      | Sensor name - VOC 8                                              | String                   |      |                                                                                                      |
+| softwareVersionBasis             | general         | Software version basis (format xx.xx)                            | String                   |      |                                                                                                      |
+| operatingHoursSupplyAirVent      | general         | Operating hours supply air fan (in minutes)                      | Number:Time              | min  | 0 - 2^32-1                                                                                           |
+| operatingHoursExtractAirVent     | general         | Operating hours extract air fan (in minutes)                     | Number:Time              | min  | 0 - 2^32-1                                                                                           |
+| operatingHoursVhz                | general         | Operating hours preheater (in minutes)                           | Number:Time              | min  | 0 - 2^32-1                                                                                           |
+| operatingHoursNhz                | general         | Operating hours afterheater (in minutes)                         | Number:Time              | min  | 0 - 2^32-1                                                                                           |
+| outputPowerVhz                   | general         | Output power of preheater (in percent)                           | Number:Dimensionless     | %    | 0 - 2^32-1                                                                                           |
+| outputPowerNhz                   | general         | Output power of afterheater (in percent)                         | Number:Dimensionless     | %    | 0 - 2^32-1                                                                                           |
+| resetFlag                        | general         | Reset flag                                                       | Switch                   |      | ON = reset flag                                                                                      |

```

Comment:
What's the purpose of this Channel?

Reply:
It resets the errors. I've adapted the description to be more expressive ;)